src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstallerServices.cs

```diff
@@ -76,7 +77,7 @@ public IEnumerable<IVsPackageMetadata> GetInstalledPackages()
 
                                 // find packages using the solution level packages folder
                                 string installPath;
-                                if (buildIntegratedProject != null)
+                                if (buildIntegratedProject != null || projectKProject != null)
```

Comment:
suggestion : 
if (project is BuildIntegratedNuGetProject || project is ProjectKNuGetProject)